src/client/app/containers/ExportContainer.ts

```diff
@@ -8,6 +8,16 @@ import ExportComponent from '../components/ExportComponent';
 import { ChartTypes } from '../types/redux/graph';
 import { ExportDataSet } from '../types/readings';
 import { State } from '../types/redux/state';
+import {CompressedBarReading, CompressedBarReadings, CompressedLineReading} from '../types/compressed-readings';
+
+
+function transformLineReadingToLegacy(reading: CompressedLineReading): [number, number] {
```

Comment:
I think `decompressLineReading` is a more semantically accurate name here.

Reply:
I kind of like legacy cause it makes it obvious that it's not a way that we like doing this anymore.

Reply:
I kind of like legacy since it really tells you that we shouldn't be using naked `[number, number]` pairs all over the place. But I'm flexible.

It's also still a compressed reading. To decompress it you'd create more than one point.